src/ProfilePage/Api/profileApi.ts

```diff
@@ -88,76 +68,61 @@ export const getStats = async (
     return 0;
   }
 
-  try {
-    const starredUrl = `https://api.github.com/users/${username}/starred`;
-    const starredData = await getPaginatedData(starredUrl);
+  const starredUrl = `https://api.github.com/users/${username}/starred`;
```

Comment:
same comment as the previous one, maybe consider having a baseUsersURL set to ` https://api.github.com/users/${username}/ ` and make reference to it